package/environments/production.js

```
@@ -1,80 +1,53 @@
 /* eslint global-require: 0 */
 /* eslint import/no-dynamic-require: 0 */
 
+const { resolve } = require("path")
 const { merge } = require("webpack-merge")
-const CompressionPlugin = require("compression-webpack-plugin")
-const TerserPlugin = require("terser-webpack-plugin")
 const baseConfig = require("./base")
 const { moduleExists } = require("../utils/helpers")
 const config = require("../config")
 
+const optimizationPath = resolve(__dirname, "..", "optimization", `${config.bundler}.js`)
```

ESLint

[error] Replace `__dirname,·"..",·"optimization",·`${config.bundler}.js`` with `⏎··__dirname,⏎··"..",⏎··"optimization",⏎··`${config.bundler}.js`⏎`

(prettier/prettier)